src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -11,6 +11,8 @@ public partial class PatchMapDrawer : Control
     [Export]
     public bool DrawDefaultMapIfEmpty;
 
+    public uint PlayerSpeciesID = int.MaxValue;
```

Comment:
Non-godot properties shouldn't be mixed in so this field should be after LineContainerPath